src/FixedDataTable.js

```diff
@@ -18,20 +18,21 @@ import FixedDataTableColumnResizeHandle from 'FixedDataTableColumnResizeHandle';
 import FixedDataTableEventHelper from 'FixedDataTableEventHelper';
 import FixedDataTableRow from 'FixedDataTableRow';
 import React from 'React';
+import {Scrollbars} from 'react-custom-scrollbars'
 import PropTypes from 'prop-types';
 import ReactTouchHandler from 'ReactTouchHandler';
 import ReactWheelHandler from 'ReactWheelHandler';
-import Scrollbar from 'Scrollbar';
 import columnTemplatesSelector from 'columnTemplates';
 import cx from 'cx';
 import debounceCore from 'debounceCore';
 import isNaN from 'lodash/isNaN';
 import joinClasses from 'joinClasses';
 import scrollbarsVisible from 'scrollbarsVisible';
 import tableHeightsSelector from 'tableHeights';
+import {SCROLLBAR_STYLE} from 'scrollbar';
 
 var ARROW_SCROLL_SPEED = 25;
-
+const SCROLL_STOP_DEBOUNCE_TIME = 50;
```

Comment:
Hey, why was this changed from 50 to 200?

Reply:
For better response to scroll stop, 200ms delay seems too much.